### ISSUE TITLE
How to follow symlinks when using modman

### ISSUE THREAD

**Issue by robbieaverill** [EcomDev/EcomDev_PHPUnit#263]
Hi,

If I install modules with modman, the paths are not followed by PHPunit.

The version of PHPunit I'm using (4.8.16) has the changes made by Sebastian here: https://github.com/sebastianbergmann/php-file-iterator/pull/21, yet I can't see the modules when I run `phpunit --list-groups`. If I target their symlinked path to the Test folder directly, they run.

If I `var_dump($modules)` in `EcomDev_PHPUnit_Test_Suite::suite()` on line 45, my symlinked module is not one of the modules returned.

Is this a Magento issue, a PHPUnit issue or an EcomDev_PHPUnit issue?

Cheers,
Robbie

**Comment by IvanChepurnyi** [comment]
Hi Robbie,

What is your directory stucture?

**Comment by robbieaverill** [comment]
Hi @IvanChepurnyi,

I've got the EcomDev PHPUnit framework installed into my main repository. I use it regularly.

I recently initiated modman and cloned a small repository I've been working from GitHub into the local pool. It now resides in `./modman/my-repo-name/app/code/local/MyVendor/MyPackage/Test`. Modman has symlinked it from `app/code/local/MyVendor` at the MyPackage point to the above (minus the Test).

FYI my example module has a `ConfigTest.php` test class under `MyPackage/Test/Config/ConfigTest.php` and a resource model test under `Test/Model/......./ColletionTest.php`. (Yeah, the phpunit declaration is in my config.xml, although it's under the symlink of MyPackage.)

Here's an example of what works and doesn't:

* `phpunit --group MyVendor_MyPackage` (no tests found)
* `phpunit --list-groups` (doesn't list MyVendor_MyPackage)
* `phpunit` (doesn't run my tests)
* `phpunit app/code/local/MyVendor/MyPackage` (doesn't run my tests)
* `phpunit app/code/local/MyVendor/MyPackage/Test` (DOES run my tests)

Hope this helps.

Cheers,
Robbie
